Description: Recognize command "icedove" as MAILER_THUNDERBIRD
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/sendto/plugins/emailclient/emailclient.c
+++ b/sendto/plugins/emailclient/emailclient.c
@@ -95,7 +95,7 @@
 		/* Find what the default mailer is */
 		if (strstr (mail_cmd, "balsa"))
 			type = MAILER_BALSA;
-		else if (strstr (mail_cmd, "thunder") || strstr (mail_cmd, "seamonkey")) {
+		else if (strstr (mail_cmd, "thunder") || strstr (mail_cmd, "seamonkey") || strstr (mail_cmd, "icedove")) {
 			char **strv;
 
 			type = MAILER_THUNDERBIRD;
